docs/getting-started-copy.md

```diff
@@ -0,0 +1,144 @@
+
+#### Installing support for Scala (Apache Toree kernel)
+
+We have tested the latest version of Apache Toree for Scala 2.11 support, and to enable that support, please do the following steps:
+
+
+* Install Apache Toree
+
+``` Bash
+# pip-install the Apache Toree installer
+pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.2.0-incubating-rc1/toree-pip/toree-0.2.0.tar.gz
+
+# install a new Toree Scala kernel which will be updated with Enterprise Gateway's custom kernel scripts
+jupyter toree install --spark_home="${SPARK_HOME}" --kernel_name="Spark 2.1" --interpreters="Scala"
+
+```
+
+* Update the Apache Toree Kernelspecs
+
+We have provided some customized kernelspecs as part of the Jupyter Enterprise Gateway releases.
+These kernelspecs come pre-configured with YARN client and/or cluster mode. Please use the steps below
+as an example on how to update/customize your kernelspecs:
+
+``` Bash
+wget https://github.com/jupyter-incubator/enterprise_gateway/releases/download/v0.6/enterprise_gateway_kernelspecs.tar.gz
+
+SCALA_KERNEL_DIR="$(jupyter kernelspec list | grep -w "spark_2.1_scala" | awk '{print $2}')"
+
+KERNELS_FOLDER="$(dirname "${SCALA_KERNEL_DIR}")"
+
+tar -zxvf enterprise_gateway_kernelspecs.tar.gz --strip 1 --directory $KERNELS_FOLDER/spark_2.1_scala_yarn_cluster/ spark_2.1_scala_yarn_cluster/
+
+cp $KERNELS_FOLDER/spark_2.1_scala/lib/*.jar  $KERNELS_FOLDER/spark_2.1_scala_yarn_cluster/lib
+```
+
+#### Installing support for Python (IPython kernel)
+
+The IPython kernel comes pre-installed.
+
+#### Installing support for R (IRkernel)
+
+```Bash
+# Perform the following steps on Jupyter Enterprise Gateway hosting system as well as all YARN workers
+
+yum install -y "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
+yum install -y git openssl-devel.x86_64 libcurl-devel.x86_64
+
+# Create an R-script to run and install packages
+cat <<'EOF' > install_packages.R
+install.packages(c('repr', 'IRdisplay', 'evaluate', 'git2r', 'crayon', 'pbdZMQ',
+                   'devtools', 'uuid', 'digest', 'RCurl', 'argparser'),
+                   repos='http://cran.rstudio.com/')
+devtools::install_github('IRkernel/IRkernel')
+IRkernel::installspec(user = FALSE)
+EOF
+
+# run the package install script
+$ANACONDA_HOME/bin/Rscript install_packages.R
+
+# OPTIONAL: check the installed R packages
+ls $ANACONDA_HOME/lib/R/library
+
+Next copy the R kernelspecs to all YARN workers
+[ ENTERPRISE_GATEWAY ] is the root directory of the JEG github repository
+cp -r [ ENTERPRISE_GATEWAY ]/etc/kernelspecs/spark_2.1_R* /usr/local/share/jupyter/kernels/
+cp -r [ ENTERPRISE_GATEWAY ]/etc/kernel-launchers/R/scripts /usr/local/share/jupyter/kernels/spark_2.1_R_yarn_client/
+cp -r [ ENTERPRISE_GATEWAY ]/etc/kernel-launchers/R/scripts /usr/local/share/jupyter/kernels/spark_2.1_R_yarn_cluster/
+
```

Comment:
The above should be replaced with reference to the `jupyter_enterprise_gateway_kernelspecs-<release>.tar.gz` as is the case for the `scala kernel` section.  No need to pull from the "JEG" repo.

Also, shouldn't the `ipython kernel` section be updated in a similar manner?

Btw, the scala section only mentions `cluster` and not `client` mode.